docs/htdocs/info/docs/tools/vep/script/vep_download.html

```diff
@@ -127,6 +127,25 @@ <h2 id="new">What's new?</h2>
 
     <h4>New in version [[SPECIESDEFS::ENSEMBL_VERSION]] <i>([[SPECIESDEFS::ENSEMBL_RELEASE_DATE]])</i></h4>
 
+    <ul>
+      <li>Added Ensembl VEP support for annotating your structural variants with allele frequencies from gnomAD and clinical significance (CLINSIG) from ClinVar.</li>
```

Comment:
```suggestion
      <li>Added Ensembl VEP support for annotating structural variants with allele frequencies from gnomAD and clinical significance (CLINSIG) from ClinVar.</li>
```

should we remove the `your`?

Reply:
Yeah, seems better